#goal definition
float32 descend_accel
float32 hit_ground_accel
---
#result definition
float32 thrust
---
#feedback
float32 thrust

